capstone-frontend/src/components/UserSearchList.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { Container, Row, Spinner } from 'react-bootstrap';
+import { UserCard } from './UserCard';
+
+export class UserSearchList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      loading: true,
+    }
+  }
+
+  getUsers = async () => {
```

Comment:
Don't think this needs to be async/awaited

Reply:
You're right, I've removed the async and await here.